bower.json

```diff
@@ -11,7 +11,7 @@
     "build/react-motion.map"
   ],
   "dependencies": {
-    "react": "^0.14.9 || ^15.3.0"
+    "react": "^15.5.0"
```

Comment:
Thanks! 15.5 would be probably too harsh, I doubt that many projects got react update. Actually our projects are still on previous 15.x release.